pkg/kv/kvserver/rangefeed/processor_test.go

```diff
@@ -268,11 +291,43 @@ func withSpan(span roachpb.RSpan) option {
 	}
 }
 
-func makeIntentScannerConstructor(rtsIter storage.SimpleMVCCIterator) IntentScannerConstructor {
-	if rtsIter == nil {
-		return nil
+type blockingScanner struct {
+	wrapped IntentScanner
+
+	block chan interface{}
```

Comment:
nit: could use a comment saying that this always blocks by default, and that it's the caller's responsibility to unblock it by closing the channel.